Stratis.Bitcoin/Features/MemoryPool/MempoolManager.cs

```diff
@@ -9,23 +9,49 @@
 
 namespace Stratis.Bitcoin.Features.MemoryPool
 {
+    /// <summary>
+    /// Memory pool scheduler.
+    /// </summary>
     public class MempoolScheduler : AsyncLock
     { }
 
+    /// <summary>
+    /// Memory pool manager.
+    /// </summary>
```

Comment:
I would appreciate long winded description about the component here. If I'm not familiar with memory pool and I read this file I have no idea what is "memory pool manager" - i.e. what is its responsibility, what it manages, what is outside of its scope. Some kind of bigger picture description would be nice about how this component fits into the architecture.